### PR TITLE
CW-832-cupcake-post-release-things

### PR DESCRIPTION
**Pull request by MrCyjaneK**
- Random name generator button
- Not entering a wallet name error (should be enhanced)
- on restore wallet top screen it says "create wallet" instead of restore wallet
- Not entering a wallet name error (should be more friendly and preferably on the field not a popup)
- When renaming a wallet to the same name, we need to improve the error message
- titles for pin screen
- tap to copy (verify - add a toast)
- biometric auth
- create pin screen - once there is a pin make only one pin, without confirmation

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -13,12 +14,14 @@ late String signingKeyFound = "";
...
-  //   signingKeyFound = e.toString();
-  // }
   await initializeBaseStoragePath();
+  if (config.initialSetupComplete == false) {
+    final oldSecureStorage = await secureStorage.readAll();
+    final date = DateTime.now().toIso8601String();
+    config.oldSecureStorage[date] = oldSecureStorage;
+    config.save();
+    await secureStorage.deleteAll();
+  }
```
Comment (OmarHatem28) [inline lib/main.dart:24]: this is for debugging only right?

Comment (MrCyjaneK) [reply lib/main.dart:24]: No this is for iOS - as it doesn't properly cleanup secure storage.

Comment (OmarHatem28) [reply lib/main.dart:24]: but I don't see any place where you set it to back false
or am I missing the purpose of this, can you explain it

File: lib/widgets/form_builder.dart [hunk 2]
```diff
@@ -29,6 +37,13 @@ class _FormBuilderState extends State<FormBuilder> {
     setState(() {});
   }
 
+  String? lastSuggestedTitle = DateTime.now().toIso8601String();
+  void onLabelChange(String? suggestedTitle) {
```
Comment (OmarHatem28) [inline lib/widgets/form_builder.dart]: having same name is confusing, I know the other one would need `widget.` but still should at least be private then

File: lib/main.dart (hunk 1 above)
Comment (MrCyjaneK) [reply lib/main.dart:24]: It is false by default, after user has completed initial setup it is set to true. Config file is located in normal app directory, that gets removed when user resets app data / reinstalls the app.

**Review by OmarHatem28 — APPROVED** [review]
biometric auth is not working
please allow enabling it from the settings instead of long press in the PIN screen, make it similar to Cake wallet
I will disable it for now, and let's make that in a separate PR with the other major features

**Pull request merged**